templates/docs/examples/patterns/side-navigation/_expand_script.js

```diff
@@ -0,0 +1,43 @@
+// Setup expand toggles for all side navigations on the page.
+
+var expandToggles = document.querySelectorAll('.p-side-navigation__expand');
+var navigationLinks = document.querySelectorAll('.p-side-navigation__link');
+
+const setup = (toggle) => {
```

Comment:
This setup part is a bit tricky depending on how backend is implemented.

If backend only sets `aria-expanded` on the button, it causes an edge case where the button may have `aria-expanded=true`, but link and list will be set to false by this setup function.

So I think ideally we want setup to:
- check if the `__expand` button has aria-expanded, if it doesn't set to false
- set aria-expanded on link and list to the same value as the expand button

This way the setup should end up with consistent state.

Reply:
Good catch! Updated.